gammapy/maps/core.py

```diff
@@ -1111,6 +1111,38 @@ def _plot_interactive(**ikwargs):
                 fig, ax, cbar = img.plot(stretch=stretch, **kwargs)
                 plt.show()
 
+    def plot_mask(self, ax=None, **kwargs):
+        """Plot the mask as a shaded area
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axis`
+            Axis instance to be used for the plot.
+        **kwargs : dict
+            Keyword arguments passed to `~matplotlib.pyplot.axvspan`
+
+        Returns
+        -------
+        ax : `~matplotlib.pyplot.Axis`
+            Axis used for plotting
+        """
+        import matplotlib.pyplot as plt
+
+        if not self.is_mask:
+            raise ValueError("This is not a mask and cannot be plotted")
+
+        kwargs.setdefault("alpha", 0.05)
+        kwargs.setdefault("colors", "k")
```

Comment:
My proposal would be to go for `color="w"` and `alpha=0.5` or similar, this will result in a plot like this:![image](https://user-images.githubusercontent.com/3715928/117664446-116b6000-b1a2-11eb-9c3e-92582163bc67.png)
Where the masked area is greyed out, which is very intuitive I think...

Reply:
Done.